### PR TITLE
MNTOR-2341: instrumentation for exiting "find exposures" screen

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-2341
Figma: https://www.figma.com/file/LGjPMlfA7Li3xkS5YZ6VLY/Attribution-Documentation?type=whiteboard&node-id=156-31791&t=6qaaSpfRCWMPPiHA-4

# Description
Instrumenting exiting "find exposures" screen 

# Screenshot (if applicable)

Not applicable.

# How to test
Get to the free scan screen, wait for the scan to complete or exit before it stops

# Checklist (Definition of Done)
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Comment by mansaj** [comment]
I'm not sure if unit tests are possible (or helpful) in this case. LMK if there's a good way to test. I've used Chrome to test the events via `GA Debug`

File: src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx [hunk 1]
```diff
@@ -67,7 +84,21 @@ export const FindExposures = ({
...
 
+  const { gtag } = useGa();
+  const pathName = usePathname();
+
   useEffect(() => {
+    // view page telemetry
+    const findExposuresTelemetryParams: FindExposuresTelemetryParams = {
+      exit_time: 0,
+      page_location: pathName,
+      user_type: "non_legacy",
```
Comment (Vinnl) [inline src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx:95]: Don't we need to detect whether the user's account is older than Plus is? AFAIK you can also get to this flow from the Guided Experience.

Comment (mansaj) [reply src/app/(proper_react)/redesign/(authenticated)/user/welcome/FindExposures.tsx:95]: Ah, thanks for the note. I assumed it's only from dashboard. It makes sense. I think we can detect this via `referrer` in the uri

**Review by Vinnl — APPROVED** [review]
Thanks for updating!

**Pull request merged**